infra/base-images/base-builder/install_python.sh

```diff
@@ -18,4 +18,5 @@
 echo "ATHERIS INSTALL"
 unset CFLAGS CXXFLAGS
 pip3 install -v --no-cache-dir "atheris>=2.0.6" "pyinstaller==4.1"
+pip3 install coverage
```

Comment:
nit: add to line 20 (and pin to version) for consistency?

Reply:
Fixed